### DISCUSSION TITLE
Extracting msw handlers to different files causes fatal error: `handler.test` is not a function

### DISCUSSION THREAD

**Discussion by misha-erm** [mswjs/msw#1013] [Q&A] [answered]
Hello,

I'm so confused. After I extracted some some of my handlers to another file they started to behave differently. I did copy-pasting but debugging in console shows me that `rest.get('', () => {})` returns different object depending on file where it's located.

Now my code looks like this

handlers.js
```
import * as blocks from './handlers/blocks';

export const handlers = [
   ...blocks.handlers
]
```

./handlers/blocks.js

```
import {rest} from "msw";

export handlers = [
  rest.get('/api/...', () => {})
]

```

Any ideas what can be the reason of it? Thanks in advance

**Comment by misha-erm** [comment, marked as answer]
For some reasons `createRestHandler` doesn't get partially applied and `rest.get` remains equal to `createRestHandler`. Maybe something wrong with my environment (webpack + babel
)

**Reply by kettanaito** [reply]
Please, could you create a reproduction repository for us to look at? I wasn't able to reproduce this, I must be missing some context.

**Reply by misha-erm** [reply]
Looks like I've figured out the problem. I have a monorepo and one of packages uses msw: 0.26.2 

For some unclear reasons my tests imported wrong version from different package.

**Comment by sheben404** [comment]
I encountered a similar problem and I solved it by running `yarn add --dev cross-env`

**Comment by jmbeach** [comment]
for me it was that I didn't spread my handlers properly. I did `server.use([...myArray])` instead of `server.use(...[...myArray])`
